### PR TITLE
New function `check_cardinality()`

### PR DESCRIPTION
**Pull request by TSchiefer**
- returns the nature of the relationship between `parent_table$pk_col` and `child_table$fk_col`

### CODE REVIEW

File: R/check-cardinalities.R [hunk 1]
```diff
@@ -125,3 +128,29 @@ check_cardinality_0_1 <- function(parent_table, pk_column, child_table, fk_colum
...
+
+#' @rdname check_cardinality
+#' @export
+check_cardinality <- function(parent_table, pk_column, child_table, fk_column) {
+  pt <- enquo(parent_table)
+  pkc <- enexpr(pk_column)
+  ct <- enquo(child_table)
+  fkc <- enexpr(fk_column)
+
+  check_key(!!pt, !!pkc)
```
[the remark this answers is not in the document]
Comment (TSchiefer) [reply R/check-cardinalities.R]: Yes, but this is intended (parent table has to have a primary key; this needs to be adapted as soon as compound keys are supported)

[the remark this answers is not in the document]
Comment (krlmlr) [reply R/check-cardinalities.R]: Before moving on, let's investigate (in a new branch) if `is_cardinality_...()` functions will make the implementation easier.

[the remark this answers is not in the document]
Comment (TSchiefer) [reply R/check-cardinalities.R]: Alright, but would you like to generally avoid the `verbose` argument, also in `check_if_subset()` and `check_set_equality()`? In this case we might need a more basic version of these two functions, which would be called by their counterparts, right? We could then use those basic functions for `is_cardinality...()`.

Is that how you intended this?

[the remark this answers is not in the document]
Comment (krlmlr) [reply R/check-cardinalities.R]: Yes, the `verbose` argument won't be necessary then.

I'm not sure why we settled for exceptions, but with the new `check_cardinality()` function it seems like we really should go for functions that check without throwing exceptions. I'd like to play with that code myself first.

[the remark this answers is not in the document]
Comment (TSchiefer) [reply R/check-cardinalities.R]: I refactored and removed the `verbose`-argument

**Pull request merged**